src/lib/components/compositions/checkbox/animated.svelte

```diff
@@ -1,22 +1,33 @@
 <script lang="ts">
+  import { afterUpdate } from 'svelte';
+
   import Error from "$components/primitives/message/error.svelte";
   import Label from "$components/primitives/label/label.svelte";
 
   export let checkValidity: ((event: Event) => boolean) | null = null;
   export let errorMessage: string = '';
+  export let firstInvalidInput: boolean;
   export let isRequired = false;
   export let isInvalid: boolean | null = null;
   export let key: string;
   export let onChange: (event: Event) => void;
   export let value: boolean;
 
+  let inputEl: HTMLInputElement;
+
   function onChangeWrapper(event: Event) {
     if (checkValidity) {
       isInvalid = !checkValidity(event);
     }
     onChange(event);
   }
 
+  afterUpdate(() => {
+    if (firstInvalidInput) {
+      inputEl.focus();
+    }
+  });
+
```

Comment:
You'll see these changes repeated throughout the components. It's what supports keyboard focus management after network request, as opposed to what you saw before, which was focus management before network request.

Reply:
Makes sense, this works nicely on the registration page if testing in the app

Reply:
Actually looks like I was talking about text input fields after network request, not sure how to test checkbox but I'm sure this makes sense

Reply:
Good point. I'm not sure if this will ever be used, but the `BooleanAttributeInputCallback`, which would use this checkbox component, can have a failed policies 🤷 

So, I have this available on this component, even though it may rarely, if ever be used.